src/Microsoft.Extensions.RuntimeEnvironment.Sources/project.json

```diff
@@ -0,0 +1,9 @@
+{
+  "version": "1.0.0-*",
+  "shared": "*.cs",
+  "dependencies": {},
+  "frameworks": {
+    "netstandard1.0": {},
+    "net451": {}
```

Comment:
I'm not sure what the TFM here should be - should we add `System.Runtime.InteropServices.RuntimeInformation` here and consequently pick the TFMs it supports? If not, targeting `netstandard1.0` should suffice.